test/src/lib.rs

```diff
@@ -395,21 +367,12 @@ mod tests {
         mark_skipped(&mut tests, "arm", "clang");
         mark_skipped(&mut tests, "mips", "clang");
         mark_skipped(&mut tests, "mipsel", "clang");
-
-        // Ghidra does not recognize all extern function calls in the disassembly step for MIPS.
-        // Needs own control flow graph analysis to be fixed.
         mark_skipped(&mut tests, "mips64", "clang");
```

Comment:
Did you find out why the clang compiled files still fail?

Reply:
For the same reason as for the ARM-based clang test cases: The constant that the check looks for is not defined in the basic block of the corresponding call. Since the check only looks in the basic block before the call, it cannot find the constant.